### ISSUE TITLE
consult

### ISSUE THREAD

**Issue by gengyanlei** [GuYuc/WS-DAN.PyTorch#7]
Hello, may you fully reproduce the accuracy of the paper on the CUB dataset? 89.4%?

**Comment by GuYuc** [comment]
@gengyanlei Hi. The discuss of reproduction is in this issue: #1 . Unluckily, we still cannot reproduce the results, but the authors have publisized their TensorFlow code in [https://github.com/tau-yihouxiang/WS_DAN/](url)
